package.json

```diff
@@ -11,10 +11,11 @@
     "migrations/scripts"
   ],
   "scripts": {
+    "build": "hardhat compile",
     "ganache": "echo 'GANACHE IS NO LONGER USED; use buidler-vm script instead'",
     "buidler-vm": "buidler node",
     "buidler-vm:log": "buidler node | grep '^    \\|console.log'",
```

Comment:
Let's remove those commands if we got rid of buidler.

Reply:
Done in https://github.com/keep-network/sortition-pools/pull/127/commits/4ba45853abf00e3486568229a9bbddac8475f0bb.